LICENSE

```diff
@@ -1,21 +1,21 @@
-MIT License
```

Comment:
There seems to be no change. Can you please revert this?

Reply:
reverted changes caused by dos2unix